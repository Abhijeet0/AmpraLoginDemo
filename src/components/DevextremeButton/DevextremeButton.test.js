import React from 'react';
import Button from 'devextreme-react/button';
import renderer from 'react-test-renderer';

// describe('DevextremeButton', () => {
//    it('renders without error', () => {
//       const tree = renderer
//       .create(<Button text = "Login"/>)
//       .toJSON();
//       expect(tree).toMatchSnapshot();
//    });
// });

// test('<Button /> renders with text as prop', () => {
//    const tree = renderer.create(
//      <Button text="Login" />
//    ).toJSON();
//    expect(tree).toMatchSnapshot();
//  });

it("renders loader component correctly", () => {
	const tree = renderer.create(<button />).toJSON();
	expect(tree).toMatchSnapshot();
});