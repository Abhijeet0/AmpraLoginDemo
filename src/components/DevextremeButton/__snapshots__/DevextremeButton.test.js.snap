// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loader component correctly 1`] = `<button />`;
